src/components/floating-panels/FloatingPanel.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode } from 'react';
+import { FlexStyle, ViewStyle } from 'react-native';
+import { Box, Space } from '@rainbow-me/design-system';
+
+type FloatingPanelProps = {
+  borderRadius: number;
+  children: ReactNode;
+  overflow: FlexStyle['overflow'];
+  paddingBottom: Space;
+  style: ViewStyle;
+  testID: string;
+};
+
+const FloatingPanel = ({
```

Comment:
@markdalgleish @jxom How do we feel about this pattern where we are passing down style props individually? It feels like the types here could get out of hand if we essentially replicate the underlying primitives style types?

Reply:
To me it feels like it's a sign that the component is too unopinionated and that it's not worth the added abstraction. This might be worth a chat, but I feel like this component either needs to be more opinionated, locking the API down to as few props and valid prop values as possible, or we're better off just inlining this Box where FloatingPanel is used instead so that consumers can see what it's doing directly.

Reply:
Im going to leave this in here for now, but I do agree with you that we should think about making this either more opinionated or just an inline component. However as its used in several places Im a bit concerned about changing it too much in this PR.